homeassistant/components/solax/manifest.json

```diff
@@ -3,7 +3,7 @@
     "name": "Solax Inverter",
     "documentation": "https://www.home-assistant.io/components/solax",
     "requirements": [
-      "solax==0.1.0"
+      "solax==0.1.1"
```

Comment:
Had to update the solax library again - `0.1.0` still had a bug where the connection to the inverter wasn't being closed correctly.